swift/Tests/CosmosSignerTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CosmosSignerTests.swift
+//  TrustWalletCoreTests
+//
+//  Created by Nan Jiang on 4/4/19.
```

Comment:
Please use the standard header.

Reply:
> Please use the standard header.

done in 06e0d0d3b5011833279af64ed0118dbcf243567c